browser/scripts/ui.js

```diff
@@ -745,14 +745,5 @@ VizorUI.checkCompatibleBrowser = function() {
 		message = '<h4>Please view this page on your desktop/laptop. '+
 					 'The editor is not ready for mobile just yet.</h4>';
 	}
-	else {
-		heading = 'Browser support';
-		message = '<h4>We want you to fully enjoy Vizor. <br />The editor works best in '+
-					 '<a href="http://www.google.com/chrome/" target="_blank"'+
```

Comment:
don't we still want to keep this warning for other browsers like Opera, IE and Edge?